File: Non Linear Data Structures/Trees/Javascript/TerneryTree.js

class Node {
    constructor(data) {
        this.data = data;
        this.left = null;
        this.mid = null;
        this.right = null;
    }
}

root = new Node(10);
root.left = new Node(11);
root.mid = new Node(12);
root.right = new Node(13)

console.log(root.data)
console.log(root.left.data)
console.log(root.right.data)

File: Non Linear Data Structures/Trees/Python/TerneryTree.py

class Node:
    def __init__(self , data):
        self.data = data
        self.left = None
        self.mid = None
        self.right = None
        
root = Node(1)
root.left = Node(2)
root.mid = Node(3)
root.right = Node(4)

print(root.data)
print(root.left.data)
print(root.mid.data)
print(root.right.data)



        